e2e-etl-python/files/Makefile

```diff
@@ -48,27 +48,7 @@ prep-test:
 .PHONY: test
 # Run (pre-deployment) tests.
 test: install-test-deps
```

Comment:
No need to define the stage if it will be empty

Reply:
The stage of the Makefile is called from the shared library, that's why if we remove it it fails. Since in this point we don't have access to the shared, we need to leave it empty.

An sample execution if you remove the test stage:
![image](https://github.com/iaizarc/ods-quickstarters/assets/140490127/21410523-884d-4b16-b837-f6004be679c0)